java-util/src/main/java/io/druid/java/util/common/granularity/Granularity.java

```diff
@@ -150,6 +148,10 @@ public int compare(Granularity g1, Granularity g2)
 
   public abstract DateTime toDate(String filePath, Formatter formatter);
 
+  public DateTime nextRound(DateTime time) {
```

Comment:
What do you think about renaming `truncate()` to `bucketStart()` and `nextRound()` to `bucketEnd()`, for more consistency with each other, and with existing `bucket()` method?